reference-app/src/main/java/org/smartregister/anc/application/AncApplication.java

```diff
@@ -200,4 +209,9 @@ public void onTimeZoneChanged() {
         context.userService().getAllSharedPreferences().saveForceRemoteLogin(true, context.allSharedPreferences().fetchRegisteredANM());
         logoutCurrentUser();
     }
+
+    @Override
+    public void onStatusUpdate(boolean b) {
+        Timber.i(this.getClass().getSimpleName(), "onStatusUpdate() by P2P");
+    }
```

Comment:
Is there a reason this method is just performing a log? Do we have to implement it?

Reply:
This has been removed